parlai/agents/seq2seq/seq2seq.py

```diff
@@ -17,10 +17,17 @@
 
 
 class Seq2seqAgent(Agent):
-    """Simple agent which uses an LSTM to process incoming text observations."""
+    """Simple agent which uses an RNN to process incoming text observations.
```

Comment:
can you give me a summary of what all the updates do?
does it support ranking now? or anything else new?